tests/test_handler_DataHandler.py

```diff
@@ -315,3 +315,12 @@ def test_writec(nexc):
     for expected in expected_patterns:
         assert re.search(expected, written, re.MULTILINE), \
             'Expected "%s"' % expected
+
+
+def test_wrapped_data(examples):
+    r = NexusReader.from_file(examples / 'example-wrapped-data.nex')
+    assert r.data.format['gap'] == '-'
+    assert len(r.data.taxa) == 3 and r.data.nchar == 471
+    sites = r.data.matrix[r.data.taxa[-1]]
+    assert len([s for s in sites if s == r.data.format['gap']]) == 5
+    assert len([s for s in sites if s == r.data.format['missing']]) == 85
```

Comment:
@SimonGreenhill the above two lines seem to show how `.format` was supposed to be used, right?

Reply:
yep!